src/main/java/io/cdap/plugin/gcp/gcs/sink/RecordFilterOutputFormat.java

```diff
@@ -132,4 +143,127 @@ public void close(TaskAttemptContext context) throws IOException, InterruptedExc
       delegate.close(context);
     }
   }
+
+  /**
+   * OutputCommitter for GCS
+   */
+  public static class GCSOutputCommitter extends OutputCommitter {
```

Comment:
Why is this needed for setting content type?

Reply:
We need to set up the content type when the job is finished. This is the same as GCS Sink plugin. However, if there's another way to do it, please let me know.

Reply:
Is `GCSOutputCommitter` exactly same as 
https://github.com/data-integrations/google-cloud/blob/develop/src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSOutputFormatProvider.java#L108?

If so why not make it a class file and use the same class at both the places?

Reply:
PR Updated.